File: app/src/main/java/com/mehmetakiftutuncu/muezzin/activities/MuezzinActivity.java
package com.mehmetakiftutuncu.muezzin.activities;

import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;

/**
 * Created by akif on 04/06/16.
 */
public abstract class MuezzinActivity extends AppCompatActivity {
    public void setTitle(String title) {
        ActionBar supportActionBar = getSupportActionBar();

        if (supportActionBar != null) {
            supportActionBar.setTitle(title);
        }
    }

    public void setTitle(int titleResId) {
        ActionBar supportActionBar = getSupportActionBar();

        if (supportActionBar != null) {
            supportActionBar.setTitle(titleResId);
        }
    }

    public void setSubtitle(String subtitle) {
        ActionBar supportActionBar = getSupportActionBar();

        if (supportActionBar != null) {
            supportActionBar.setSubtitle(subtitle);
        }
    }

    public void setSubtitle(int subtitleResId) {
        ActionBar supportActionBar = getSupportActionBar();

        if (supportActionBar != null) {
            supportActionBar.setSubtitle(subtitleResId);
        }
    }
}

File: app/src/main/java/com/mehmetakiftutuncu/muezzin/activities/MuezzinActivity.kt
package com.mehmetakiftutuncu.muezzin.activities

import android.content.Context
import androidx.appcompat.app.AppCompatActivity

abstract class MuezzinActivity : AppCompatActivity() {
    protected val ctx: Context by lazy { this }

    fun setTitle(title: String) =
        supportActionBar?.apply {
            this.title = title
        }

    override fun setTitle(titleResId: Int) {
        supportActionBar?.apply {
            setTitle(titleResId)
        }
    }

    fun setSubtitle(subtitle: String) =
        supportActionBar?.apply {
            this.subtitle = subtitle
        }

    fun setSubtitle(subtitleResId: Int) {
        supportActionBar?.apply {
            setSubtitle(subtitleResId)
        }
    }
}